README.md

```diff
@@ -62,6 +64,27 @@ export github_personal_token=[paste access token here]
 
 You should then run `source ~/.bash_profile` to reload the settings.
 
+## Setup using Docker
```

Comment:
Should this be a subheading in the general 'Setup' section above? Also we should clearly explain at the beginning of 'Setup' that there are two setup options: non-Docker (i.e. the legacy way) and Docker. If readers follow the legacy Setup instructions and _then_ see the Docker Setup - and want to do that instead - they will have done the legacy setup for no reason.